.github/workflows/w3.yml

```diff
@@ -0,0 +1,30 @@
+name: w3 cli
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'packages/w3/**'
+      - '.github/workflows/w3.yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'packages/w3/**'
+      - '.github/workflows/w3.yml'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Test
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      # Only install the deps directly listed... dont do workspace magic.
+      # Running install from the root gets you all the deps for all the packages.
+      # which can easily mask missing deps if another modules also depends on it.
+      - run: npm install
+        working-directory: packages/w3
+      - run: npm test
+        working-directory: packages/w3
```

Comment:
@yusefnapora thanks for the nudge. You may be pleased to learn that we're now testing in CI to make sure that all the deps are declared!

Reply:
nice!